install/0000_90_cluster-version-operator_00_servicemonitor.yaml

```diff
@@ -16,3 +16,42 @@ spec:
   selector:
     matchLabels:
       k8s-app: cluster-version-operator
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    k8s-app: cluster-version-operator
+  name: cluster-version-operator
+  namespace: openshift-cluster-version
+spec:
+  groups:
+  - name: cluster-version
+    rules:
+    - alert: ClusterVersionOperatorDown
+      annotations:
+        message: ClusterVersionOperator has disappeared from Prometheus target discovery.
+      expr: |
+        absent(up{job="cluster-version-operator"} == 1)
+      for: 15m
+      labels:
+        severity: critical
+  - name: cluster-operators
+    rules:
+    - alert: ClusterOperatorDown
+      annotations:
+        message: ClusterOperator {{ $labels.name}} has not been available for 10 mins.
```

Comment:
I would prefer to check for degraded or not available.  Both are significant.

Reply:
Also explain impact here.

Reply:
> I would prefer to check for degraded or not available.

Isn't that what [the current `available && !failing`][1] is doing?  If you're not available, we're not setting `cluster-operator-up` for you.  If you're available but degraded (which backs the `failing` variable), we're not setting it either.

[1]: https://github.com/openshift/cluster-version-operator/commit/72e8f1a9d1c51d2fbb62ce4b61de37b183298489#diff-b301ca75c2e59524b215636eb7a4586aR252

Reply:
added `ClusterOperatorDegraded` in https://github.com/openshift/cluster-version-operator/pull/232/commits/48237bc2395f66194c99017b84406ce70aee8256